lang/python/python-dateutil/Makefile

```diff
@@ -15,33 +15,51 @@ PKG_LICENSE:=BSD-2-Clause
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/python-dateutil
 PKG_HASH:=88f9287c0174266bb0d8cedd395cfba9c58e87e5ad86b2ce58859bc11be3cf02
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-dateutil-$(PKG_VERSION)
 
+include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
+
 include ../python-package.mk
+include ../python3-package.mk
 
-define Package/python-dateutil
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+HOST_UNPACK:=$(HOST_TAR) -C $(HOST_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/python-dateutil/Default
   SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
   MAINTAINER:=Gergely Kiss <mail.gery@gmail.com>
   TITLE:=Extensions to the standard Python datetime module
   URL:=https://dateutil.readthedocs.org/
+endef
+
+define Package/python-dateutil
+$(call Package/python-dateutil/Default)
   DEPENDS:=+python +python-six
```

Comment:
the reverse is valid here as in comment https://github.com/openwrt/packages/pull/7899/commits/c38f4b20e6691a838f4b84b88fefc24c148f0c95#r245613285
you can add `DEPENDS:=+PACKAGE_python-dateutil:python +PACKAGE_python-dateutil:python-six`

this would avoid building Python + python-dateutil if you only want the Python3 versions (i.e. python3-dateutil)

there is an exception though; if this creates circular dependencies which show up if you run `make menuconfig` or `make defconfig` then the `PACKAGE_python-dateutil:` prefix should be skipped.

Reply:
done